rholang/src/main/scala/coop/rchain/rholang/interpreter/storage/StoragePrinter.scala

```diff
@@ -88,6 +88,7 @@ object StoragePrinter {
               receiveBinds,
               p.body,
               persist,
+              peeks.nonEmpty,
```

Comment:
1) I don't think we need to be so defensive in the printer given that the current syntax doesn't allow it
2) Yeah, makes sense - added

Reply:
If the syntax doesn't allow that - what's wrong with throwing the exception? The point of being defensive is so that we spot things immediately when the outside context / callsite changes. I insist :)

Reply:
A better question would be "if the syntax doesn't allow that, what's wrong with not throwing the exception"? :-)
If you insist though, perhaps a better place to throw would be `WaitingContinuation.create`?